docs/BEST_PRACTICES.md

```diff
@@ -0,0 +1,10 @@
+## Deployment best practices
+
+Both modules should be deployed as a publicly exposed service.
+
+Modules are **stateless** which makes running multiple separate instances in parallel possible.
+It is recommended to **enable horizontal scaling**
+with at least 2 running instances at the same time in order to minimize downtime
+possibility.
+
+It's also recommended to use HTTPS.
```

Comment:
Maybe lets make bullet point out of every best practice?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/45/commits/040b94c792033fb105093402aa1b3d7a2b980348